osquery/sql/sqlite_util.cpp

```diff
@@ -223,6 +223,32 @@ class StringEscaperVisitor : public boost::static_visitor<> {
   }
 };
 
+class SizeVisitor : public boost::static_visitor<> {
+ public:
+  void operator()(const long long& i) {
+    size = sizeof(i);
+  }
+
+  void operator()(const double& d) {
+    size = sizeof(d);
+  }
+
+  void operator()(const std::string& t) {
+    size = t.length();
+  }
+
+  uint64_t get_size() const {
+    return size;
+  }
+
+  SizeVisitor() {
+    size = 0;
+  }
+
+ private:
+  uint64_t size;
+};
```

Comment:
nit: prefer
```suggestion
private:
  uint64_t size{0};
};
```
or `SizeVisitor() : size(0)`